en/views/helpers/form.rst

```diff
@@ -1022,6 +1022,27 @@ methods are described in each method's own section.)
           'hiddenField' => 'N',
       ]);
 
+Using Collections to build options
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It's possible to use the Collection class to build your options array. This approach is ideal if you already have a
+collection of entities and would like to build a select element from them.
+
+You can use the ``combine`` method to build a basic options array.
+
+    $options = $entitiesArray->combine('id', 'name');
```

Comment:
Change `$entitiesArray` to `$entities` or `$articles`? Looks cleaner, looks more like real world naming.

Reply:
It also makes the content and type of the variable unknown.

Reply:
That's right. But still, you usually do `$users = $this->Users->find()->all()` rather than `$usersArray`. A small detail, but think it harmonizes better with the other examples in the docs and real world naming.